src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+
+/**
+ * Represents the ExitCommand to exit Focus.
+ */
+public class ExitCommand extends Command {
+    /**
+     * Returns true since ExitCommand is not an ExitCommand.
```

Comment:
I think you had a typo here. It should be is an ExitCommand.

Reply:
Oh right, thank you!